### PR TITLE
User Meeting banner for home page

### PR DESCRIPTION
**Pull request by hflynn**
Still to do:

- [x] Add link to registration site
- [x] Add registration details to event page
- [x] Announce registration is open

Waiting on link from June at the moment

### CODE REVIEW

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/index.html

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/02/23/user-meeting-registration.html and https://snoopycrimecop.github.io/www.openmicroscopy.org/events/13th-annual-users-meeting-2018.html as well as the homepage

File: _posts/2018-02-23-user-meeting-registration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+We are pleased to announce that registration for the 2018 OME Annual Users Meeting is now open. Book your place at [Buyat Dundee](https://www.buyat.dundee.ac.uk/conferences-and-events/school-of-life-sciences/ome-annual-users-meeting/2018-ome-annual-users-meeting).
+
+The meeting will be held May 30th - June 1st 2018 at the University of Dundee,
+Scotland, UK.
+
+Registration is open until 30th April 2018 and is £115 per person, including
+lunches, refreshments, dinners on 30th and 31st May, and airport transfers by
+bus if required.
+
+You will be invited to give a lightning talk or poster during the registration
```
Comment (mtbc) [inline _posts/2018-02-23-user-meeting-registration.md]: ... or present a poster ... ?  (Just a suggestion.)

File: css/openmicroscopy.css [hunk 2]
```diff
@@ -243,7 +243,7 @@ h3.subdivider {
     background-repeat: no-repeat;
     background-size: cover;    
 }
-
+
```
Comment (mtbc) [inline css/openmicroscopy.css]: probably a mistake, but harmless

**Comment by mtbc** [comment]
Thank you, looks good to me.

**Comment by will-moore** [comment]
What links to registration at https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/02/23/user-meeting-registration.html?
The home page "register now" links to https://www.buyat.dundee.ac.uk/conferences-and-events/school-of-life-sciences/ome-annual-users-meeting/2018-ome-annual-users-meeting

**Comment by hflynn** [comment]
The Buyat Dundee link is on that announcement too @will-moore but I guess the wording is different, I'll unify it

**Comment by will-moore** [comment]
How do users get to this page?
https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/02/23/user-meeting-registration.html
Is there a link to it anywhere?

**Comment by hflynn** [comment]
It's in the announcements feed - https://snoopycrimecop.github.io/www.openmicroscopy.org/announcements/

**Comment by hflynn** [comment]
Snoopy pages updated

**Comment by will-moore** [comment]
OK, thanks

**Comment by joshmoore** [comment]
Various thumbs up. Releasing to ome-www.

**Pull request merged**